guidelines/sc/20/visual-presentation.html

```diff
@@ -14,6 +14,8 @@ <h4>Visual Presentation</h4>
      <li>Text can be resized without assistive technology up to 200 percent in a way that does not require the user to scroll horizontally to read a line of text <a>on a full-screen window</a>.</li>   
   </ul>
 
-<p class="note">The 'mechanism' for achieving the presentation can be provided by the user-agent, which can manipulate the content on behalf of the user. It is not a requirement for the author to provide the mechanism. Some writing systems have additional or different requirements, or do not customarily use features such as paragraph spacing. When a specific feature is inapplicable to a writing system, this criterion does not require a mechanism to achieve that feature for content authored in that writing system.</p>
+<p class="note">Content is not required to use these values. The requirement is that a mechanism is available for users to change these presentation aspects. The mechanism can be provided by the browser or other user agent. Content is not required to provide the mechanism.</p>
+
+<p class="note">Writing systems for some languages use different presentation aspects to improve readability and legibility, such as paragraph start indent. If a presentation aspect in this success criterion is not used in a writing system (for example, it does not use paragraph spacing), content in that writing system can conform without it.</p>
```

Comment:
This gives permission to not use a "presentation aspect" that doesn't apply, but doesn't give permission **to use** one that is not listed, which I think we wanted to encourage. I'd suggest:

```suggestion
<p class="note">Writing systems for some languages might use presentation aspects not present in this success criterion to improve readability and legibility or the language might not use one of the presentation aspects that are present. Nothing in this success criterion should be interpreted as restricting access to additional presentation aspects nor requiring content to use a specific presentation aspect that would not normally be applied to a language.</p>
```

Note that I took the specific examples out (these probably belong in understanding). I also didn't use "Users of writing systems..." for brevity.

Reply:
> I took the specific examples out 

OK.

> I think we wanted to encourage

I pulled wording from https://github.com/w3c/wcag/pull/3437 :
> Authors are encouraged to follow guidance for improving readability and legibility of text in their writing system.

I really think we want to keep the concepts, language, and sentence structure as simple as feasible. e.g., I think "Writing systems for some languages use different presentation..." is simple and factual. I'm not sure we need to say more here. Again, can say lots more in the Understanding docs.

Reply:
@aphillips How about with the tweaks now in https://github.com/w3c/wcag/pull/3439/files ?